lxd/container_lxc.go

```diff
@@ -4245,7 +4272,12 @@ func (c *containerLXC) State() string {
 		return "BROKEN"
 	}
 
-	return c.c.State().String()
+	cString := "Error"
+	state, err := c.GetLxcState()
+	if err == nil {
+		cString = state.String()
+	}
```

Comment:
Also, as you defined the new state in shared/ shouldn't .String() actually work here?